browser/eb_application.py

```diff
@@ -150,17 +150,19 @@ def load(filename):
 
     folders = re.split('__', filename)
     filename = folders[len(folders) - 1]
-    subfolders = folders[2:len(folders)]
+    subfolders = folders[2:len(folders)-1]
```

Comment:
The linter must not be set up.

Reply:
Correct, no specific linter set up. I can spot fix this one (not sure which part of it needs linting in particular) and we can look into more thorough linting later?

Reply:
Yea that can be a different PR but lets keep the current changes lint-appropriate too.